### PR TITLE
Update FreeBSD docs

### PR DESCRIPTION
**Pull request by lwhsu**

### CODE REVIEW

File: docs/guide/FreeBSD.md [hunk 1]
```diff
@@ -20,11 +20,12 @@ task:
...
-* `freebsd-12-0`
-* `freebsd-11-3-snap` (stable `freebsd-11-3` doesn't boot properly at the moment)
-* `freebsd-11-2`
-* `freebsd-10-4`
+* `freebsd-13-0-snap` (13.0-CURRENT)
+* `freebsd-12-1-snap` (12.1-STABLE)
+* `freebsd-12-1`      (12.1-RELEASE)
+* `freebsd-12-0`      (12.0-RELEASE)
+* `freebsd-11-3-snap` (11.3-STABLE)
+* `freebsd-11-3`      (11.3-RELEASE, doesn't boot properly at the moment)
```
Comment (RDIL) [inline docs/guide/FreeBSD.md]:
```suggestion
```
lets not mention it then

Comment (fkorotkov) [reply docs/guide/FreeBSD.md]: IMO worth to mention since people did try this particular image.

Comment (lwhsu) [reply docs/guide/FreeBSD.md]: Yeah, and we're tackling it now: https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=242303

Comment (RDIL) [reply docs/guide/FreeBSD.md]: 👍

**Pull request merged**